zkevm-circuits/src/evm_circuit/util/math_gadget/lt_word.rs

```diff
@@ -84,7 +84,8 @@ mod tests {
         fn configure_gadget_container(cb: &mut EVMConstraintBuilder<F>) -> Self {
             let a = cb.query_word32();
             let b = cb.query_word32();
-            let ltword_gadget = LtWordGadget::<F>::construct(cb, &a, &b);
+            let ltword_gadget =
+                LtWordGadget::<F, Word32Cell<F>, Word32Cell<F>>::construct(cb, a, b);
```

Comment:
You could consider writing it like this, I think it should work:
```suggestion
            let ltword_gadget =
                LtWordGadget::<F, _, _>::construct(cb, a, b);
```